src/main/java/supertracker/parser/Parser.java

```diff
@@ -196,6 +203,9 @@ public static Command parseCommand(String input) throws TrackerException {
         case REVENUE_COMMAND:
             command = parseRevenueCommand(params);
             break;
+        case RENAME_COMMAND:
+            command = parseRenameCommand(params);
+            break;
         case EXPENDITURE_COMMAND:
             command = parseExpenditureCommand(params);
             break;
```

Comment:
I think it would be neater if the exp and rev commands are next to each other
```suggestion
        case RENAME_COMMAND:
            command = parseRenameCommand(params);
            break;
        case EXPENDITURE_COMMAND:
            command = parseExpenditureCommand(params);
            break;
        case REVENUE_COMMAND:
            command = parseRevenueCommand(params);
            break;
```